facility-manager.html

```diff
@@ -0,0 +1,806 @@
+
+<p>Administrators and Group Owners can change group permissions between Private, Read-Only and Read-Annotate. Only Administrators can change a group to Read-Write. Full details of permissions can be seen in the <a href="http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-permissions.html" target="_blank" class="external-url">System Administrator documentation</a>.</p>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">If permissions are reduced to Private when one member has created output from another member's image in the group using a script, e.g. made a projection, the permissions change will fail.</p>
+</div><!-- End note -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="default"></a>Default groups</h2>
+
+<p>Most facilities use a default group that a new user is automatically allocated to when they first log on and create an account using the LDAP or similar authentication system, or when an account is created for them by the Admin. Typically this group would be Private.</p>
+
+<p>If the user is also made a member of their lab or other groups, they can set any group as their default group. This group will be then shown in the data tree each time they log in.</p>
+
+<p>For details on how users can change their default group see Changing your default group in the <a href="http://help.openmicroscopy.org/sharing-data.html#default" target="_blank" class="external-url">Sharing Data</a> section of User Help.</p>
+
+<p>Administrators can set or change the default group for a user through the Admin tab in OMERO.web or the Administration tab in OMERO.insight.</p>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Although the Administration functions are shown for both OMERO.web and OMERO.insight, the OMERO.web Admin tools are recommended for ease of use over the OMERO.insight Administration tools.</p>
+</div><!-- End note -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li>
+
+<p>Click on <span class="bold">Admin</span> in the top toolbar.</p>
+
+<p>In the <span class="bold">Users</span> tab, find user by filtering.</p>
+
+<p>Click on the <span class="bold">Edit</span> button.</p>
+
+<p>Select the new default group from the drop-down list.</p>
+
+<p>Click <span class="bold">Save</span> to save the new default group.</p>
```

Comment:
Not a big fan of "click save to save" wording. 'Click to save the..." would read better, we don't need to tell people that a 'save' button saves something.